kvrocks.conf

```diff
@@ -603,7 +603,7 @@ rocksdb.max_background_flushes -1
 # This value represents the maximum number of threads that will
 # concurrently perform a compaction job by breaking it into multiple,
 # smaller ones that are run simultaneously.
-# Default: 2 (i.e. no subcompactions)
+# Default: 2
```

Comment:
The default value here is 1, according to the source code: https://github.com/apache/kvrocks/blob/unstable/src/config/config.cc#L172